README.md

```diff
@@ -49,6 +49,7 @@ If you have time and willing to help, there are a lof of ways to contribute:
  - [solr](https://github.com/netdata/go.d.plugin/tree/master/modules/solr)
  - [springboot2](https://github.com/netdata/go.d.plugin/tree/master/modules/springboot2)
  - [tengine](https://github.com/netdata/go.d.plugin/tree/master/modules/tengine)
+ - [vsphere](https://github.com/whalehub/go.d.plugin/tree/master/modules/vsphere)
```

Comment:
wrong link

Reply:
@ilyam8 My bad, pushed another commit to fix this.